README.md

```diff
@@ -10,16 +10,17 @@ advcpmv-0.5-8.21.patch was the last patch released by the author (on February 14
 
 ## Build instructions
 
-The latest GNU Core Utilities source can be found here: https://ftp.gnu.org/gnu/coreutils/
+Run the following command which will download, patch, then compile coreutils, producing two files: `./advcpmv/advcp` and `./advcpmv/advmv`.
```

Comment:
> Run the following command which will download, patch, then compile coreutils, producing two files: `./advcpmv/advcp` and `./advcpmv/advmv`.

Run the following command to download, patch, compile coreutils and generate the files: `./advcpmv/advcp` and `./advcpmv/advmv`.